ui-tests/staticSites.spec.js

```diff
@@ -0,0 +1,209 @@
+import { expect, test } from "@playwright/test";
+import dotenv from "dotenv";
+import mongoose from "mongoose";
+import categoryModel from "../models/categoryModel";
+import productModel from "../models/productModel";
+
+dotenv.config();
+
+const categorySlug = "test-category-zz";
+const productSlug = "test-product-zz";
+
+test.beforeAll(async () => {
+  // Connecting to db which is needed for creating category and product for test
+  await mongoose.connect(process.env.MONGO_URL);
+
+  const newCategory = await new categoryModel({
+    name: "Test Category ZZ",
+    slug: categorySlug,
+  }).save();
+
+  await new productModel({
+    name: "Test Product ZZ",
+    slug: productSlug,
+    description: "This is the test product zz",
+    price: 1,
+    category: newCategory._id,
+    quantity: 1,
+    shipping: true,
+  }).save();
+});
+
+test.afterAll(async () => {
+  // Delete category created from this test
+  await categoryModel.deleteOne({ slug: categorySlug });
+
+  // Delete product created from this test
+  await productModel.deleteOne({ slug: productSlug });
+
+  // Close the connection with Mongo DB since the test has finished running
+  await mongoose.disconnect();
+});
+
+const adminRoutes = [
```

Comment:
small suggestion, maybe we can use these adminRoutes and non-admin routes as a dictionary of strings for the below test cases